From 7005407fa898f3b17aaf1cb85b2372a062cdf4df Mon Sep 17 00:00:00 2001
From: Anton Kindestam <antonki@kth.se>
Date: Sun, 7 Jul 2019 13:17:55 +0200
Subject: [PATCH 1/2] libbsd-overlay

---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1b499fe..63a2740 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,8 @@
 #INCLUDE = -I/usr/local/include
 #LIB = -L/usr/local/lib
 #DEFS = -DDEBUG
-LDADD = -lsndio
+CFLAGS = $(shell pkg-config --cflags libbsd-overlay)
+LDADD = -lsndio $(shell pkg-config --libs libbsd-overlay)
 
 PREFIX = /usr/local
 BIN_DIR = ${PREFIX}/bin
-- 
2.21.0

